DotNetRu.Clients.Portable/ViewModel/MeetupsViewModel.cs

```diff
@@ -34,6 +38,7 @@ public class MeetupsViewModel : ViewModelBase
         public MeetupsViewModel(INavigation navigation)
             : base(navigation)
         {
+            MessagingCenter.Subscribe<LocalizedResources>(this, MessageKeys.LanguageChanged, sender => SortEvents());
```

Comment:
Why do we need to sort events here?!

Reply:
@bruce-willis events are just meetups (so please rename the SortEvents) method - and they are sorted by date.